### PR TITLE
feat: add datahub track

### PR DESCRIPTION
**Pull request by cbernat**
Add data hub track for current page.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ history.listen((location) => {
   const locationPage = location.hash && location.hash[0] === '#' && location.hash.slice(1);
   ReactGA.set({ page: locationPage });
   ReactGA.pageview(locationPage);
-  // TODO: Also notify navigation to DataHub (DataHub does not have this feature yet)
+  _dha.track({ navigatedPage: locationPage }); // eslint-disable-line
```
Comment (andresmoschini) [inline src/index.js]: What is the disabled rule?

Comment (andresmoschini) [reply src/index.js]: Maybe you could do something like this:

```js
   window._dha && window._dha.track({ navigatedPage: locationPage });
```

Comment (cbernat) [reply src/index.js]: To prevent eslint from throwing a warning, because `_dha` is a global variable not defined in `index.js`

**Comment by cbernat** [comment]
I will merge this, I've tested manually and rechecked with Leo from DataHub.

**Pull request merged**